<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelSetASMSegmentationView</class>
 <widget class="QWidget" name="LevelSetASMSegmentationView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QmitkDataStorageComboBox" name="ImageSelector"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>FMImage</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QmitkDataStorageComboBox" name="FMImageSelector"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>MeanImage</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QmitkDataStorageComboBox" name="MeanImageSelector"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="title">
       <string>PCA Image</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>361</width>
         <height>171</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="AddPCAImageBtn">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="RemovePCAImageBtn">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="PCAImageList"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QmitkPointListWidget" name="PointListWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ApplyBtn">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header>QmitkDataStorageComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkPointListWidget</class>
   <extends>QWidget</extends>
   <header>QmitkPointListWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
